.github/labeler.yml

```diff
@@ -1,6 +1,6 @@
 # Add/remove 'ep:ACL' label if issue contains the word 'ACL'
 ep:ACL:
-  - '(ACL)'
+  - '(/ACL/i)'
```

Comment:
I think this should be `/ACL/i`

The parenthesis are for regex grouping and we don't need that for this tool AFAIK